lib/davinci_dtr_test_kit/client_groups/dinner_static/dtr_dinner_questionnaire_package_request_test.rb

```diff
@@ -11,39 +11,73 @@ class DTRDinnerQuestionnairePackageRequestTest < Inferno::Test
-                             description: %(
-                               Patient instance id to be provided by Inferno as the `patient` as a part of the SMART app
-                               launch.
-                             )
-    input :smart_fhir_context, optional: true, title: 'SMART App Launch fhirContext (Dinner Static)',
-                               type: 'textarea',
-                               description: %(
-                               References to be provided by Inferno as the `fhirContext` as a part of the SMART app
-                               launch. These references help determine the behavior of the app. Referenced instances
-                               may be providedin the "EHR-available resources" input.
-                             )
-    input :ehr_bundle, optional: true, title: 'EHR-available resources (Dinner Static)', type: 'textarea',
-                       description: %(
-                         Resources available from the EHR needed to drive the dinner static workflow.
-                         Formatted as a FHIR bundle that contains resources, each with an `id` property populated. Each
-                         instance present will be available for retrieval from Inferno at the endpoint
-                         `[fhir-base]/[resource type]/[instance id].`
-                       )
+    input :launch_uri,
+          optional: true,
+          description: 'Required if "Launch from Inferno" is selected'
+    input :smart_patient_id,
+          optional: true,
+          title: 'SMART App Launch Patient ID (Dinner Static)',
+          type: 'text',
+          description: %(
+            Patient instance id to be provided by Inferno as the `patient` as a part of the SMART app
+            launch.
+          ),
+          default: 'pat015'
+    input :smart_fhir_context,
+          optional: true,
+          title: 'SMART App Launch fhirContext (Dinner Static)',
+          type: 'textarea',
+          description: %(
+            References to be provided by Inferno as the `fhirContext` as a part of the SMART app
+            launch. These references help determine the behavior of the app. Referenced instances
+            may be providedin the "EHR-available resources" input.
+          ),
+          default: JSON.pretty_generate([{ reference: 'Coverage/cov015' },
+                                         { reference: 'DeviceRequest/devreqe0470' }])
+    input :ehr_bundle,
+          optional: true,
+          title: 'EHR-available resources (Dinner Static)',
+          type: 'textarea',
+          description: %(
+            Resources available from the EHR needed to drive the dinner static workflow.
+            Formatted as a FHIR bundle that contains resources, each with an `id` property populated. Each
+            instance present will be available for retrieval from Inferno at the endpoint
+            `[fhir-base]/[resource type]/[instance id].`
```

Comment:
```suggestion
            instance present will be available for retrieval from Inferno at the endpoint:
            
            ```
            [fhir-base]/[resource type]/[instance id]
            ```
```